.github/workflows/main.yml

```diff
@@ -144,7 +144,7 @@ jobs:
       shell: bash
     - run: rustup target add aarch64-pc-windows-msvc
     - run: cargo test --no-run --target aarch64-pc-windows-msvc
-    - run: cargo test --no-run --target aarch64-pc-windows-msvc --features verify-winapi
+    - run: cargo test --no-run --target aarch64-pc-windows-msvc --features verify-windows-sys
```

Comment:
...I have left it as-is until recently, but why do we not simply run this all the time when we know we're on Windows?